tests/docker/universum_test_env/Dockerfile

```diff
@@ -1,20 +1,20 @@
-FROM ubuntu:xenial
+FROM ubuntu:bionic
```

Comment:
Why?

Reply:
because we need python3.7 which is missed in *xenial* apt repository by default